tests/Crypto/AesGcmEncryptingStreamTest.php

```diff
@@ -30,6 +30,11 @@ public function testStreamOutputSameAsOpenSSL(
         StreamInterface $plainText,
         $keySize
     ) {
+        if (version_compare(PHP_VERSION, '7.1', '<')) {
```

Comment:
Should there be a separate test for `AesGcmDecryptingStreamTest` and `AesGcmEncryptingStreamTest` that covers the < PHP 7.1 case (which is now supported with the `AesGcm` class implementation)?

Reply:
There is. This is just an OpenSSL compatibility test with random inputs, which isn't possible to run on PHP <7.1.